app/common/auth/password-reset-confirm.controller.js

```diff
@@ -1,9 +1,9 @@
 module.exports = PasswordResetConfirmController;
 
-PasswordResetConfirmController.$inject = ['$rootScope', 'PasswordReset', '$location', '$routeParams'];
-function PasswordResetConfirmController($rootScope, PasswordReset, $location, $routeParams) {
+PasswordResetConfirmController.$inject = ['$rootScope', 'PasswordReset', '$location', '$transition$'];
+function PasswordResetConfirmController($rootScope, PasswordReset, $location, $transition$) {
     var $scope = $rootScope.$new();
-    $scope.token = $routeParams.token;
+    $scope.token = $transition$.params().token;
```

Comment:
Did you figure out the difference between this and `$uiRouterGlobals.params`? I read the docs but can't figure out when to use which...

Reply:
I believe in general $transition$ would be the most appropriate when available since it already represents the current transition data instead of grabbing it from $current on uiRouterGlobal? but I might be wrong.